manim/mobject/shape_matchers.py

```diff
@@ -10,6 +10,23 @@
 
 
 class SurroundingRectangle(Rectangle):
+    """A rectangle surrounding a `mobject`
+
+    Examples
+    --------
+
+    .. manim:: SurroundingRectExample
+        :save_last_frame:
+
+        class SurroundingRectExample(Scene):
+            def construct(self):
+                title = Title("A Quote from Newton")
+                quote = Text("If I have seen further than others, \nit is by standing upon the shoulders of giants.", color=BLUE).scale(0.75)
```

Comment:
```suggestion
                quote = Text("If I have seen further than others, \\nit is by standing upon the shoulders of giants.", color=BLUE).scale(0.75)
```

Reply:
The way the doc-making process works, one backslash is taken away. So adding another should fix it and all checks should pass now.

Reply:
Thanks for pointing that out.